src/main/resources/logback.xml

```diff
@@ -0,0 +1,31 @@
+<configuration>
+  <property name="QUERYRESULT_LOG_FILE" value="queryResultLog" />
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${QUERYRESULT_LOG_FILE}</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+      <!-- daily rollover -->
+      <fileNamePattern>${QUERYRESULT_LOG_FILE}.%d{yyyy-MM-dd}.gz</fileNamePattern>
+      <maxHistory>12</maxHistory>
+      <totalSizeCap>3GB</totalSizeCap>
+    </rollingPolicy>
+    <append>true</append>
+    <encoder>
+      <pattern>%d{"yyyy-MM-dd'T'HH:mm:ss.SSS"};%msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="info">
+    <appender-ref ref="STDOUT"/>
+  </root>
+
+  <logger name="de.numcodex.feasibility_gui_backend.query.result.ResultService" level="INFO">
```

Comment:
@michael-82  - Query results should not log to stdout - consider using trace for ResultService file logger